win32/initnet.c

```diff
@@ -14,21 +14,33 @@
 
 HINSTANCE hinstFreeTDS;
 
+#if defined(_MSC_VER) && defined(_DEBUG)
+HANDLE h_crt_file = _CRTDBG_FILE_STDOUT;
+#endif
+
 BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved);
 
 BOOL WINAPI
 DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved TDS_UNUSED)
 {
+	const char* crt_filename = NULL;
+
 	hinstFreeTDS = hinstDLL;
 	switch (fdwReason) {
 	case DLL_PROCESS_ATTACH:
 #if defined(_MSC_VER) && defined(_DEBUG)
+		crt_filename = getenv("TDS_DEBUG_CRT_FILENAME");
```

Comment:
Better to use `tds_dir_getenv`, `tds_dir_char` instead of `char` and `CreateFileW` instead of `CreateFile`.